tests/pytorch_pfn_extras_tests/onnx/test_export_testcase.py

```diff
@@ -62,6 +62,7 @@ def _helper(model, args, d, **kwargs):
     return output_dir
 
 
+@pytest.mark.filterwarnings("ignore::UserWarning")
```

Comment:
An experimental warning for named tensors is raised, but this test looks not to use named tensor :thinking:
```
UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /pytorch/c10/core/TensorImpl.h:1156.)
```